src/op5/auth/Auth.php

```diff
@@ -548,12 +548,14 @@ public function session_destroy() {
 		if(PHP_SAPI == 'cli') return;
 		if (session_id() !== '') {
 			$name = session_name();
-			@session_destroy();
```

Comment:
So if I get it right you don't need `session_destroy` because you clear the session data with `$_SESSION = array();` below, and then remove the cookie after that?

Reply:
Yes, that's correct.
From the docs:
> Note: You do not have to call session_destroy() from usual code. Cleanup $_SESSION array rather than destroying session data.